src/NServiceBus.Testing.Fakes/NServiceBus.Testing.Fakes.csproj

```diff
@@ -12,6 +12,7 @@
   </ItemGroup>
 
   <ItemGroup>
+    <PackageReference Include="Obsolete.Fody" Version="4.4.3" />
```

Comment:
So, even though this will bring in a version of Fody, we should be explicit here and not rely on transitive dependencies.

We also should be using the same versions of Fody and Obsolete.Fody throughout the solution, so we should use the same versions as NServiceBus.Core:

```
<PackageReference Include="Fody" Version="3.1.4" PrivateAssets="All" />
<PackageReference Include="Obsolete.Fody" Version="4.4.1" PrivateAssets="All" />
```

Alternatively, we should update Core to the latest of both packages instead.